framework/core/tests/integration/api/groups/ListTest.php

```diff
@@ -153,9 +153,9 @@ public function paginates_groups_without_filter()
     {
         $response = $this->send(
             $this->request('GET', '/api/groups')
-            ->withQueryParams([
-                'page' => ['limit' => '2', 'offset' => '2'],
-            ])
+                ->withQueryParams([
+                    'page' => ['limit' => '2', 'offset' => '2'],
+                ])
```

Comment:
nitpick: just to keep the git history clean
```suggestion
            ->withQueryParams([
                'page' => ['limit' => '2', 'offset' => '2'],
            ])
```